packages/admin/cms-admin/src/dam/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -0,0 +1,144 @@
+                        fileUrl
+                    }
+                }
+            `,
+            variables: { id: output.fileId },
+        });
+
+        const { damFile } = data;
+
+        ret.file = {
+            id: damFile.id,
+            name: damFile.name,
+            fileUrl: damFile.fileUrl,
+        };
+
+        return ret;
+    },
+
+    definesOwnPadding: true,
+
+    AdminComponent: ({ state, updateState }) => {
+        return (
+            <>
+                <BlocksFinalForm<{
+                    openFileType: DamFileDownloadLinkBlockData["openFileType"];
+                    file?: DamFileDownloadLinkBlockData["file"];
+                }>
+                    onSubmit={(newValues) => {
+                        updateState({
+                            file: newValues.file ?? undefined,
+                            openFileType: newValues.openFileType,
+                        });
+                    }}
+                    initialValues={{
+                        file: state.file,
+                        openFileType: state.openFileType ?? "DOWNLOAD",
+                    }}
+                >
+                    {state.file !== undefined ? (
+                        <AdminComponentPaper disablePadding>
+                            <Box padding={3}>
+                                <Typography variant="subtitle1">{state.file.name}</Typography>
+                                <Typography variant="body1" color="textSecondary">
+                                    <DamPathLazy fileId={state.file.id} />
+                                </Typography>
+                            </Box>
+                            <Divider />
+                            <AdminComponentButton startIcon={<Delete />} onClick={() => updateState({ file: undefined, openFileType: "NEW_TAB" })}>
+                                <FormattedMessage {...messages.empty} />
+                            </AdminComponentButton>
+                        </AdminComponentPaper>
+                    ) : (
+                        <Field name="file" component={FileField} fullWidth />
+                    )}
+
+                    <Divider />
+                    <AdminComponentPaper>
+                        <Field
+                            name="openFileType"
+                            fullWidth
+                            label={<FormattedMessage id="blocks.damFileDownloadLink.openFileType" defaultMessage="Open File Type" />}
+                        >
+                            {(props) => (
+                                <>
+                                    <FinalFormSelect {...props}>
+                                        {openFileTypeOptions.map((item, index) => (
+                                            <MenuItem value={item.value} key={index}>
+                                                {item.name}
+                                            </MenuItem>
+                                        ))}
```

Comment:
Since there are only two options, I'd just hardcode them instead of extracting them to an array